### PR TITLE
clarify the value to use for some image config env variables

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation ptal

**Comment by bparees** [comment]
@openshift/team-documentation bump

File: using_images/s2i_images/nodejs.adoc [hunk 1]
```diff
@@ -78,7 +78,7 @@ section] of the build configuration's `*sourceStrategy*` definition.
 | Variable name | Description
 
 |`*DEV_MODE*`
-| Enables hot deploy, and opens debug port. Additionally, indicates to tooling that the image is in development mode. To enable, set to true. Default is false.
+| When set to `true`, enables hot deploy and opens the debug port. Additionally, indicates to tooling that the image is in development mode. To enable, set to true. Default is false.
```
Comment (bfallonf) [inline using_images/s2i_images/nodejs.adoc]: @bparees The second true and the false needs some markup (`true`).

**Comment by bfallonf** [comment]
@bparees One comment, but other than that, LGTM.

**Comment by bparees** [comment]
@bfallonf updated.

**Comment by bfallonf** [comment]
@bparees Cool. LGTM. I'll give this a merge.

File: using_images/s2i_images/nodejs.adoc [hunk 2]
```diff
@@ -78,7 +78,7 @@ section] of the build configuration's `*sourceStrategy*` definition.
 | Variable name | Description
 
 |`*DEV_MODE*`
-| Enables hot deploy, and opens debug port. Additionally, indicates to tooling that the image is in development mode. To enable, set to true. Default is false.
+When set to `true`, enables hot deploy and opens the debug port. Additionally, indicates to tooling that the image is in development mode.  Default is `false`.
```
Comment (adellape) [inline using_images/s2i_images/nodejs.adoc]: @bparees Think there's a missing pipe here.

**Comment by adellape** [comment]
@bfallonf Was the `to_follow` label just about markup?

File: using_images/s2i_images/nodejs.adoc (hunk 2 above)
Comment (bparees) [inline using_images/s2i_images/nodejs.adoc]: thanks, was having vi issues.  fixed.

Comment (adellape) [inline using_images/s2i_images/nodejs.adoc]: :+1:

**Comment by adellape** [comment]
[rev_history]
|link:../using_images/s2i_images/nodejs.html[Source-to-Image (S2I) -> Node.js]
|Clarified description of the `*DEV_MODE*` environment variable.
%
|link:../using_images/s2i_images/nodejs.html[Source-to-Image (S2I) -> Node.js]
|Clarified description of the `*DEV_MODE*` environment variable.
%|link:../using_images/s2i_images/nodejs.html[Source-to-Image (S2I) -> Node.js]
|Clarified description of the `*DEV_MODE*` environment variable.
%

**Comment by adellape** [comment]
Just realized this needs https://github.com/openshift/openshift-docs/pull/1395 (which still needs follow-up) before cherry-picking.

**Comment by adellape** [comment]
Still trying to verify whether the changes described in https://github.com/openshift/openshift-docs/pull/1395 actually made it into the nodejs image currently used in 3.1.

**Comment by bparees** [comment]
@adellape i don't see them in the current nodejs rhel image, so i'm going to say no, they'll go in as part of 3.2.

**Comment by adellape** [comment]
@bparees Thank you

**Comment by bfallonf** [comment]
@adellape Followup edits to #1395 have merged. Can we remove the followup label here or is there anything else to catch?

**Comment by adellape** [comment]
@bfallonf Think it's good now! I'll remove it.

**Pull request merged**